mne/inverse_sparse/mxne_inverse.py

```diff
@@ -347,6 +348,16 @@ def mixed_norm(evoked, forward, noise_cov, alpha, loose='auto', depth=0.8,
     %(pick_ori)s
     %(verbose)s
 
+    grid_lower_bound : float
```

Comment:
I would add a single new parameter called sure_alpha_grid which is set to "auto" by default and the docstring says what "auto" converts to.